libs/cardano-ledger-test/src/Test/Cardano/Ledger/Examples/BabbageFeatures.hs

```diff
@@ -344,7 +344,8 @@ referenceScriptTx pf =
       WitnessesI
         [ AddrWits' [makeWitnessVKey (hashAnnotated (referenceScriptTxBody pf)) (someKeys pf)],
           DataWits' [datumExample1],
-          RdmrWits validatingRedeemersEx1
```

Comment:
Have Jared look over these

Reply:
I pushed a commit, we can removed these added script witnesses, which I've done after fixing a problem with how script references were being gather.